### PR TITLE
OBPIH-3615 Add invoice view page

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

**Review by pmuchowski — APPROVED** [review]

**Comment by DominikaHatala** [comment]
@jmiranda Please look at this one and merge it if everything is fine today.

File: grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy [hunk 1]
```diff
@@ -106,6 +112,27 @@ class Invoice implements Serializable {
         return getReferenceNumber(Constants.VENDOR_INVOICE_NUMBER_TYPE_ID)
     }
 
+    Organization getVendor() {
```
Comment (jmiranda) [inline grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy:115]: Same with these, I'd prefer an association.

File: grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy [hunk 2]
```diff
@@ -106,6 +112,27 @@ class Invoice implements Serializable {
...
+        return Organization.get(party?.id)
+    }
+
+    Organization getBuyerOrganization() {
```
Comment (jmiranda) [inline grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy:119]: Same with these, I'd prefer an association.

**Review by jmiranda — APPROVED** [review]

**Comment by jmiranda** [comment]
@DominikaHatala yeah i was in the process of reviewing it. sorry for the delay ... had a few meetings since we spoke. Check out my comments. Please create a new ticket under the invoice epic to add those associations and the related database migrations.

**Comment by DominikaHatala** [comment]
@jmiranda I haven't seen any comments about those associations and also didn't find it in Invoice Data Model in Confluence. Sure I can create ticket.

**Pull request merged**